lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -172,7 +181,7 @@ export class ResponseHandler {
                 env,
                 serverTokenResponse.access_token,
                 this.clientId,
-                idTokenObj.claims.tid,
+                idTokenObj ? idTokenObj.claims.tid : authority.tenant,
```

Comment:
`authority.tenant` could be `common` right? Is that ok?

Reply:
I think we cache with `common` but we should verify this.

Reply:
Yes, it would just be cached as common.